app/src/main/java/org/mozilla/focus/utils/DebugUtils.java

```diff
@@ -17,12 +17,23 @@ private DebugUtils() {
     }
 
     public static String loadWebViewVersion(Context context) {
-        return loadWebViewVersion(new WebView(context));
+        String webViewVersion;
```

Comment:
Will
```
        try {
            return loadWebViewVersion(new WebView(context));   
        } catch (Exception ex) {
            return "";
        }
```
be simpler?

Reply:
It's kinda a habit for debugging purpose. Do you agree?